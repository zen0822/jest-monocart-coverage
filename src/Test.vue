<template>
  <div>
    {{ props.className }}
  </div>
</template>

<script setup lang="ts">
const props = defineProps({
  className: {
    type: String,
    default: '',
  },
})
</script>
