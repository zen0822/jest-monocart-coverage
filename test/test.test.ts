import { mount } from '@vue/test-utils'

import Test from '../src/Test.vue'

describe('测试 Test.vue', () => {
  test('View', async () => {
    const wrapper = await mount(Test, {
      props: { className: 'className' },
    })

    expect(wrapper.html()).toMatchSnapshot()
  })
})
