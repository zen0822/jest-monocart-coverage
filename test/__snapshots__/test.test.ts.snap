// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`测试 Test.vue View 1`] = `"<div>className</div>"`;
